journalpump/journalpump.py

```diff
@@ -1000,7 +1000,9 @@ def process(self):
             else:
                 new_entry[key.lstrip("_")] = value
 
-        self.reader.perform_searches(self.jobject)
+        if self.reader.searches:
```

Comment:
This should probably be after the `if self.jobject.cursor is None` check?

Reply:
I did that initially, but decided to keep the change as minimal as possible. I can change it if you'd like, I don't think it makes much of a difference in practice.